### PR TITLE
feature/8887-HighLevelClaimsDetailsViews

### PR DESCRIPTION
**Pull request by Sparowhawk**
## Description of Change
All of this work is behind a feature flag for it to display(since the feature flag hasn't been created yet I have a hard coded boolean for true, prior to merging I will flip this false or replace it with the feature flag once it is created)

Now for the changes:
1. Changed the title from bitter bold variant to mobile body bold and adjusted the spacing between the title and the received on date
2. Changed the need help section to be always present regardless of segment control status and adjusted spacing between the body and the links
3. Cleaned up some non needed testIdPropsUsage
4. Included Remote config setup/alphabetization even though it isn't explicitly used in this ticket, but will in future tickets

## Screenshots/Video
Before

Screenshot 2024-06-26 at 12 20 41 PM
Screenshot 2024-06-26 at 12 20 49 PM

After

Screenshot 2024-06-26 at 12 10 33 PM
Screenshot 2024-06-26 at 12 10 38 PM
Screenshot 2024-06-26 at 12 10 43 PM
Screenshot 2024-06-26 at 12 10 47 PM
Screenshot 2024-06-26 at 12 11 29 PM
Screenshot 2024-06-26 at 12 11 32 PM
Screenshot 2024-06-26 at 12 11 42 PM

## Testing
yarn test

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
Update claim type header style [claim type]
 Show Need help section across both (Active and Closed) segments
 Typography aligns with with [Figma mockup](https://www.figma.com/design/kSlXLwTVLunM0fqCG95k5x/%F0%9F%94%8D-Claims-2.0---Working---VA-Mobile?node-id=1586-1774&t=Kz9shxOuIFDRYddG-4)
 Need help section repeats across each segment, and aligns with Figma / [Copy doc](https://docs.google.com/spreadsheets/d/16waC665d7uNRQmM83SJIm9bTDyRePOuB6r9G_aQE8Tw/edit?gid=728579729#gid=728579729&range=A23)
 VQA with Lauren / Misty

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [x] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [x] No secrets or API keys are checked in
- [x] All imports are absolute (no relative imports)
- [x] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

**Comment by alexandec** [comment]
For the feature flag, I think you can add it the same way I did in this PR I just created: https://github.com/department-of-veterans-affairs/va-mobile-app/pull/8928

**Comment by Sparowhawk** [comment]
I will just wait for your list pr to be merged first then just use the one you created

**Comment by Sparowhawk** [comment]
@alexandec since your ticket got moved back to in progress I just duplicated what you did for those screens

File: VAMobile/src/utils/remoteConfig.ts
Comment (alexandec) [inline VAMobile/src/utils/remoteConfig.ts:64]: I put claimPhaseExpansion last in my PR so that will probably conflict with these. Though probably we should alphabetize them at this point, which would help avoid conflicts too

File: VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.tsx
Comment (alexandec) [inline VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.tsx:126]: Probably can just be {showButton}

**Review by alexandec — DISMISSED** [review]
Approving with a couple optional suggestions

File: VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.tsx
Comment (Sparowhawk) [reply VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimDetailsScreen/ClaimStatus/ClaimStatus.tsx:126]: updated

File: VAMobile/src/utils/remoteConfig.ts
Comment (Sparowhawk) [reply VAMobile/src/utils/remoteConfig.ts]: alphabetized

**Review by alexandec — DISMISSED** [review]

**Comment by Sparowhawk** [comment]
Added same header and need help panel change for Appeals, see thread in slack for screenshots

**Review by alexandec — APPROVED** [review]

**Review by rbontrager — APPROVED** [review]

**Pull request merged**
